x-pack/plugin/transform/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformPivotRestSpecialCasesIT.java

```diff
@@ -248,8 +248,8 @@ public void testRestrictiveBucketSelector() throws Exception {
         String indexName = "special_pivot_bucket_selector_reviews";
         createReviewsIndex(indexName, 1000, 327, "date", false, 5, "affiliate_id");
 
-        verifyDestIndexHitsCount(indexName, "special_pivot_bucket_selector-10", 10, 14);
-        verifyDestIndexHitsCount(indexName, "special_pivot_bucket_selector-10000", 10000, 14);
+        verifyDestIndexHitsCount(indexName, "special_pivot_bucket_selector-10", 10, 41);
```

Comment:
The change here is the result of fixing the geo-coordinates that were failing bulk upload